### PR TITLE
Fade in QualityInfo while hovering on Song title 

### PR DESCRIPTION
**Pull request by shishir610**
This PR addresses - 
1. Fade-in animation of QualityInfo while hovering on the Song title
2. Anchor tag covering issue by changing height of the audio player

Video demonstrates both. 

https://user-images.githubusercontent.com/27198773/115147662-ae424e00-a079-11eb-908b-c571df95147b.mp4

Signed-off-by: Shishir <shishir.srik@gmail.com>

### CODE REVIEW

File: ui/src/common/QualityInfo.js [hunk 1]
```diff
@@ -33,12 +34,14 @@ export const QualityInfo = ({ record, size, className }) => {
   }
 
   return (
-    <Chip
-      className={clsx(classes.chip, className)}
-      variant="outlined"
-      size={size}
-      label={info}
-    />
+    <Fade in={songTitleHover} timeout={500}>
```
Comment (deluan) [inline ui/src/common/QualityInfo.js:37]: The component should not incorporate Fade, as it is not its concern. For example, now in the SongGrid and AlbumSongs, the quality does not appear at all! Please move this logic to the AudioTitle component

File: ui/src/audioplayer/Player.js [hunk 2]
```diff
@@ -62,17 +70,28 @@ const AudioTitle = React.memo(({ audioInfo, isMobile }) => {
   return (
     <Link to={`/album/${audioInfo.albumId}/show`} className={className}>
       <span className={`${className} songTitle`}>
-        {audioInfo.name}
+        <span
+          onMouseEnter={() => setSongTitleHover(true)}
```
Comment (deluan) [inline ui/src/audioplayer/Player.js:74]: I rather not do this with JavaScript. Can you find a solution with pure CSS? Ex: the context menu and heart are toggled with pure CSS

**Review by deluan — CHANGES_REQUESTED** [review]
The end result is great, but I think we need to change the implementation. See my comments below.

File: ui/src/audioplayer/Player.js [hunk 3]
```diff
@@ -37,10 +37,24 @@ const useStyle = makeStyles(
...
+    audioName: {
+      '&:hover + $qualityInfo': {
+        opacity: 1,
+      },
     },
     player: {
       display: (props) => (props.visible ? 'block' : 'none'),
+      '& .music-player-panel': {
+        height: '100px',
+        paddingBottom: '10px',
```
Comment (deluan) [inline ui/src/audioplayer/Player.js:53]: I think making the player taller is worse than having the link appearing on top of the progress bar when you are actually hovering a link. I prefer to not do this.

**Review by deluan — CHANGES_REQUESTED** [review]
Thanks, looks great, but please check my comment below.

File: ui/src/audioplayer/Player.js (hunk 3 above)
Comment (shishir610) [reply ui/src/audioplayer/Player.js:53]: Alright, so it's okay that the link appears on top of the progress bar?

Comment (deluan) [reply ui/src/audioplayer/Player.js:53]: Yes, it is ok.

IMHO, if you are hovering a link, it is expected you will have an address showing in the bottom, and you are probably not interested in seeing the progress bar at that moment.

Comment (shishir610) [reply ui/src/audioplayer/Player.js:53]: Yeah, that makes sense, I'll remove it and commit

**Comment by deluan** [comment]
Thanks!

**Pull request merged**